app/src/main/java/me/phh/treble/app/Misc.kt

```diff
@@ -188,8 +188,19 @@ object Misc: EntryStartup {
                 SystemProperties.set("persist.sys.fflag.override.settings_fuse", if (!value) "true" else "false")
             }
             MiscSettings.backlightScale -> {
-                val value = sp.getBoolean(key, false)
-                SystemProperties.set("persist.sys.phh.backlight.scale", if (value) "1" else "0")
+                val value = sp.getString(key, "System Default")
+                android.util.Log.d("PHH", "Setting backlight scale to $value")
+                when (value) {
+                  "System Default" -> { 
```

Comment:
again, have you tested? Considering your pref_misc.xml, I'm expecting those values to be all lower case